virtcontainers/container.go

```diff
@@ -1131,6 +1133,10 @@ func (c *Container) hotplugDrive() error {
 		return nil
 	}
 
+	if dev.mountPoint == c.rootFs {
+		c.rootfsSuffix = ""
```

Comment:
Would it be possible to contrive a unit test for the two possible values here?

Reply:
I saw the unit test that got added. But I feel we would be better served with a system level test here. :(

Reply:
the function `hotplugDrive` does a lot of system/OS level calls to identify  the characteristic of the drive that is being handed for rootfs. To changes i made to code to unit test this, does not sit well with me, but i was unable to find another cleaner way to do this test without doing a full blown device creation and mocking a few more things. Could you check the code and let me know your thoughts. This case i think is one that might benefit with more functional tests against unit tests.